tests/garage/experiment/test_deterministic.py

```diff
@@ -0,0 +1,17 @@
+import torch
+
+from garage.experiment import deterministic
+
+
+def test_deterministic_pytorch():
```

Comment:
omg nice. can you add one for TF while you're here?

Reply:
and numpy and random.random, as well?

Reply:
Yes, I added them as well.